javascriptExercises/page01-81Exercises/recursion.js

```diff
@@ -0,0 +1,14 @@
+const isEven = function(even){
+    if (even == 0){
+        return true;
+    }
+    else if(even == 1 || even < 0) {
+        return false;
+    }
+    else{
```

Comment:
Please use consistent spacing. There is a space between the first if and the `(` sign, but secon id and else words don't have spaces after them